qemu/tests/blockdev_inc_backup_inc_success.py

```diff
@@ -0,0 +1,121 @@
+from provider import backup_utils
+from provider import blockdev_base
+from provider import block_dirty_bitmap
+
+from virttest import utils_misc
+
+
+class BlockdevIncbkIncSyncSuccBitmapTest(blockdev_base.BlockdevBaseTest):
+
+    def __init__(self, test, params, env):
+        super(BlockdevIncbkIncSyncSuccBitmapTest, self).__init__(
+            test, params, env)
+        self.source_images = []
+        self.full_backups = []
+        self.inc_backups = []
+        self.inc_backup_tags = []
+        self.bitmaps = []
+        self.src_img_tags = params.objects("source_images")
+        self.inc_sync_mode = params["inc_sync_mode"]
+        self.inc_bitmap_mode = params["inc_bitmap_mode"]
+        list(map(self._init_arguments_by_params, self.src_img_tags))
+
+    def _init_arguments_by_params(self, tag):
+        image_params = self.params.object_params(tag)
+        image_chain = image_params.objects("image_backup_chain")
+        self.source_images.append("drive_%s" % tag)
+        self.full_backups.append("drive_%s" % image_chain[0])
+        self.inc_backups.append("drive_%s" % image_chain[1])
+        self.inc_backup_tags.append(image_chain[1])
+        self.bitmaps.append("bitmap_%s" % tag)
+
+    def do_full_backup(self):
```

Comment:
I thinks it can be merged to one function with do_incremental_backup with different extra_options..